src/templates/static/affiliates/signup.jsx

```diff
@@ -1,31 +1,211 @@
 import React from 'react';
-import FAQ from './faq.jsx';
-import Main from './main.jsx';
+import { Table } from '../../_common/components/elements.jsx';
+import { TabContainer, TabContent, TabContentContainer, TabsSubtabs } from '../../_common/components/tabs.jsx';
+
+const BoxRow = ({ children, top_row, bottom_row }) => (
+    <div className={`box-row gr-row ${top_row ? 'box-row-top' : ''} ${bottom_row ? 'box-row-bottom' : ''}`}>
+        {children}
+    </div>
+);
+
+const Box = ({ image, title, text }) => (
+    <div className='box gr-4 gr-12-m center-text gr-padding-30'>
+        <img className='gr-centered fixed-height-img' src={it.url_for(`images/pages/affiliates/${image}.svg`)} alt={image} />
+        <p><strong>{title}</strong></p>
+        <p>{text}</p>
+    </div>
+);
+
+const Step = ({ header, text, circle_no }) => (
+    <div className='step'>
+        <div className='border-bottom' />
+        <div className='circle'>{ circle_no }</div>
+        <div className='gr-padding-20 gr-gutter center-text'>
+            <div className='gr-padding-20 gr-child'><strong>{header}</strong></div>
+            <p className='no-margin gr-padding-10'>{text}</p>
+        </div>
+    </div>
+);
 
 const Signup = () => (
-    <div className='static_full'>
-        <h1>{it.L('Affiliate')}</h1>
-
-        <div className='content-tab-container gr-padding-10'>
-            <div className='tab-menu'>
-                <div className='tab-menu-wrap container'>
-                    <ul className='tm-ul'>
-                        <li id='general_info' className='tm-li active first'><span className='menu-wrap-a'><span className='menu-wrap-b'><span className='tm-a a-active'>{it.L('Affiliate')}</span></span></span></li>
-                        <li id='faq' className='tm-li last'><a href='#faq' className='tm-a'>{it.L('FAQ')}</a></li>
-                    </ul>
+    <div className='static_full affiliates'>
+        <div className='container'>
+            <h1>{it.L('Binary.com Affiliate Programme')}</h1>
+            <p>
+                {it.L('Earn up to 35% commission with an award-winning binary options trading platform.')}
+            </p>
+
+            <div className='gr-padding-20'>
+                <h2 className='center-text margin-bottom-50'>{it.L('How it works')}</h2>
+                <div className='steps'>
+                    <Step circle_no='1' header={it.L('Sign up')}            text={it.L('Getting started is free and easy –– just <a href=\'[_1]\'>fill out the application form</a> and wait for our approval.', it.affiliate_signup_url)} />
```

Comment:
instead of `––` use a single dash `-` because in browser it translates to two dashes instead of a long dash `--`